library/spdm_common_lib/libspdm_com_msg_log.c

```diff
@@ -11,7 +11,9 @@ void libspdm_init_msg_log (void *context, void *msg_buffer, size_t msg_buffer_si
 {
     libspdm_context_t *spdm_context;
 
-    LIBSPDM_ASSERT((context != NULL) && (msg_buffer != NULL));
+    if (context == NULL || msg_buffer == NULL) {
```

Comment:
why we need check "context == NULL" here?

If we added check in init(), then that will guarantee all rest function calls will have context != NULL.

Please clarify in which path, you can still trigger NULL context input.